RELEASE_NOTES.md

```diff
@@ -26,6 +26,7 @@ Bumped minimum version of pandas and numpy to fit **ixmp4**'s requirement.
 
 ## Individual updates
 
+- (#879)[https://github.com/IAMconsortium/pyam/pull/879] Add read_netcdf() function
```

Comment:
This has to go to the top of this file. These are the individual changes for a previous release, not the latest version.